test/sentry/integrations/oban/cron_test.exs

```diff
@@ -4,111 +4,157 @@ defmodule Sentry.Integrations.Oban.CronTest do
 
   import Sentry.TestHelpers
 
-  setup_all do
-    Sentry.Integrations.Oban.Cron.attach_telemetry_handler()
-  end
-
-  setup do
-    bypass = Bypass.open()
+  describe "default configuration" do
```

Comment:
There seems to be only one additional test from a quick glance; if so, can we not wrap all of this in a `describe` to avoid the huge diff, and only add the new test?

Reply:
So the tests were admittedly where I felt the least comfortable changing; so I'm not surprised you'd like changes.

The problem that I was trying to address is that I assumed we need a different `setup` block in order to show that all of the previous tests remained unchanged.  Before `setup_all` was being used to attach the telemetry handler a single time for all test cases.  The describe blocks were meant to separate the cases where the telemetry handler was attached with a non-default config.

I'd be happy to update this, but I'm not exactly sure how to best proceed.

Thanks :pray:

Reply:
Well, _rubber duckie_ ... I wasn't aware that tags are a part of the context that is available in the setup function (https://hexdocs.pm/ex_unit/1.12/ExUnit.Case.html#module-tags)

I think I see a way forward.  I'll push something up for your review.